rlw.sh

```diff
@@ -98,7 +98,7 @@ rwinetricks () {
 roblox-install () {
 	printf '%b\n' " > begin roblox-install ()\n---"
 	if [[ ! -d "$WINEPREFIX/drive_c" ]]; then
-		spawndialog question 'A working Roblox wineprefix was not found.\nWould you like to install one?'
+		spawndialog question 'Roblox Linux Wrapper is not setup yet\nAre you sure you would like to install it?'
```

Comment:
Please keep the reference to the Wineprefix, for the sake of clarity. Some people would like to know this!